LanguageFeatures/FinalizationRegistry/ffi/WeakReference/expando_key_t04.dart

```diff
@@ -26,7 +26,6 @@ main() {
     Expect.throws(() {
       S s = p1.ref;
       WeakReference wr = WeakReference<S>(s);
-      print(wr.target);
```

Comment:
> Without this `wr` variable became unused (never read) and may be removed by optimizer

All we need in this test is to call `WeakReference` constructor and check that it throws with the given (ffi) argument.

Indeed, it's possible that `wr` variable assignment will not work here because `wr` is unused and so assignment itself can be optimized. But `WeakReference` constructor **must** be called anyway here.

To minimize the code, it even will be better to change the code above to the following:
      S s = p1.ref;
     Expect.throws(() {
         WeakReference<S>(s);
    }

Reply:
Agree. Please do the change